corehq/apps/hqwebapp/static/app_manager/less/form_editing.less

```diff
@@ -36,12 +36,16 @@
 }
 
 .btn-form-action {
+  border: 1px solid white;
+  padding: 5px 10px;
+  margin-right: 10px;
```

Comment:
also, if you remove button group, it's likely you will not need this

Reply:
please read up on `btn-group` to understand what it is doing. https://getbootstrap.com/docs/3.3/components/#btn-groups

the natural state of inline buttons is that buttons in a row will have some margin between them. the padding should default to the normal button padding, so that should also be removed here as well. the only thing I can see that's necessary is the border.